### PR TITLE
Add/menus

### LINKED ISSUE #166: Add `Flyout Menus` examples
### Subject

`Flyout Menus`

### Description

We need to add `Flyout Menus` examples. You may refer to https://tailwindui.com/components/marketing/elements/flyout-menus when making these examples.

### Are you willing to participate in the development of this feature and create a pull request?

None

### PR DESCRIPTION
**Pull request by illionillion**
Closes #166

### CODE REVIEW

File: contents/overlay/menus/menu-with-icon/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "en": {
+    "title": "MenuWithIcon",
+    "description": ""
```
Comment (cidkumagai) [inline contents/overlay/menus/menu-with-icon/metadata.json:4]: Please write English Description.

File: contents/overlay/menus/menu-with-icon/metadata.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  },
+  "ja": {
+    "title": "MenuWithIcon",
+    "description": ""
```
Comment (cidkumagai) [inline contents/overlay/menus/menu-with-icon/metadata.json:8]: Please write Japanese title and description.

File: contents/overlay/menus/simple-menu/metadata.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "en": {
+    "title": "SimpleMenu",
+    "description": ""
```
Comment (cidkumagai) [inline contents/overlay/menus/simple-menu/metadata.json:4]: Please write English Description.

File: contents/overlay/menus/simple-menu/metadata.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  },
+  "ja": {
+    "title": "SimpleMenu",
+    "description": ""
```
Comment (cidkumagai) [inline contents/overlay/menus/simple-menu/metadata.json:8]: Please write Japanese title and description.

File: contents/overlay/menus/simple-with-descriptions/metadata.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+{
+  "en": {
+    "title": "SimpleWithDescriptions",
+    "description": ""
```
Comment (cidkumagai) [inline contents/overlay/menus/simple-with-descriptions/metadata.json:4]: Please write English description.

**Review by cidkumagai — CHANGES_REQUESTED** [review]

File: contents/overlay/menus/simple-with-descriptions/metadata.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
+{
+  "en": {
+    "title": "Simple With Descriptions",
+    "description": "menu with descriptions"
+  },
+  "ja": {
+    "title": "Simple With Descriptions",
```
Comment (cidkumagai) [inline contents/overlay/menus/simple-with-descriptions/metadata.json:7]: please write Japanese.

**Review by cidkumagai — CHANGES_REQUESTED** [review]

**Review by cidkumagai — APPROVED** [review]
LGTM!

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**